Chefster.csproj

```diff
@@ -8,6 +8,7 @@
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <UserSecretsId>d70ac473-6c10-438a-a3ba-2a154bdf5946</UserSecretsId>
```

Comment:
So this ID is just to identify a certain users secrets to use when pulled locally? Its not sensitive right?

Reply:
this ID is just to identify the dotnet project out of all dotnet projects on your machine, non-sensitive